src/newSeed/newSeed.ts

```diff
@@ -19,7 +22,7 @@ export class NewSeed {
     return this.router.currentInstruction.config;
   }
 
-  constructor() {
+  constructor(private eventAggregator: EventAggregator) {
```

Comment:
Please use wizardState, not events for managing wizard-scoped, non-seedConfig data.

Reply:
You can set this state upon arrival in Stage 7